gdx/src/com/badlogic/gdx/graphics/g3d/utils/DefaultTextureBinder.java

```diff
@@ -66,8 +70,13 @@ public DefaultTextureBinder (final int method, final int offset, final int count
 	}
 
 	public DefaultTextureBinder (final int method, final int offset, int count, final int reuseWeight) {
-		final int max = Math.min(getMaxTextureUnits(), MAX_GLES_UNITS);
-		if (count < 0) count = max - offset;
+		int max = Math.min(getMaxTextureUnits(), MAX_GLES_UNITS);

```

Comment:
Should let final no?

Reply:
Yeah, but it's probably not worth the effort to change back.